test/aqua/test_amplitude_estimation_circuitfactory.py

```diff
@@ -325,20 +337,17 @@ class TestProblemSetting(QiskitAquaTestCase):
 
     def setUp(self):
         super().setUp()
-        warnings.filterwarnings(action="ignore", category=DeprecationWarning)
         self.a_bernoulli = BernoulliAFactory(0)
         self.q_bernoulli = BernoulliQFactory(self.a_bernoulli)
         self.i_bernoulli = 0
 
         num_qubits = 5
         self.a_integral = SineIntegralAFactory(num_qubits)
-        self.q_intergal = QFactory(self.a_integral, num_qubits)
+        with warnings.catch_warnings():
+            warnings.filterwarnings('ignore', category=DeprecationWarning)
+            self.q_intergal = QFactory(self.a_integral, num_qubits)
```

Comment:
I assume this should be spelled `q_integral` but since it is most likely consistent in the use of the spelling it has maybe its not a real issue. Same as `i_intergal` below

Reply:
Good catch! Updated in 9be5f45